modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -386,26 +354,30 @@ public Date getModifiedDate() {
 
 	@Override
 	public String getScopeName(Locale locale) {
-		return Optional.ofNullable(
-			_group
-		).map(
-			group -> {
-				try {
-					return Optional.ofNullable(
-						group.getDescriptiveName(locale)
-					).orElseGet(
-						() -> group.getName(locale)
-					);
-				}
-				catch (PortalException portalException) {
-					_log.error(portalException);
+		if (_group == null) {
+			return StringPool.BLANK;
+		}
 
-					return group.getName(locale);
-				}
+		String scopeName = null;
+
+		try {
+			scopeName = _group.getDescriptiveName(locale);
```

Comment:
@slnn In the original logic, the try-catch is only meant to catch the exception declared by `group.getDescriptiveName`. And `group.getName()` will never return null.

So you logic can be simplified as

```
try {
    scopeName = xxxx;
}
catch (E e) {
    xxxxx;
}

if (scopeName == null) {
    return _group.getName(locale);
}

return scopeName;
```